plugins/ethereum/etherscan_balance_

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python
```

Comment:
Most of my comments above apply to this plugin as well, since they share a lot of code.
Maybe you want to combine both plugins into a [multigraph plugin](http://guide.munin-monitoring.org/en/latest/plugin/multigraphing.html)?
Just put a line like the following in front of every graph (the rest can remain unchanged):
```
multigraph ethermine_balance_0x3257bde8cf067ae6f1ddc0e4b140fe02e3c5e44f
```
Or maybe use a wildcard plugin?
We want to avoid duplicate code in the repository, since it is hard to maintain.